ui/pages/confirmation/confirmation.js

```diff
@@ -271,6 +272,16 @@ export default function ConfirmationPage({
     ///: END:ONLY_INCLUDE_IF
   ];
 
+  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+  if (
+    [...Object.values(SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES)].includes(
+      pendingConfirmation?.type,
+    )
+  ) {
+    useSnapHeader = false;
```

Comment:
why are we setting `useSnapHeader` to false here when we are explicitly pushing the SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES values into the `SNAP_DIALOG_TYPE` on line 252-256? 

```js
  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
  SNAP_DIALOG_TYPE.push(
    ...Object.values(SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES),
  );
  ///: END:ONLY_INCLUDE_IF
```

If we don't want to show the snap header I think we can just remove those conformations types from this list?

Reply:
Also, maybe I am confused but it seems like this component is still being rendered when we create/remove an account...?

Reply:
To make the styling of the screen easier I disabled the header in the confirmation page itself and added it to the page implementations themselves. This made the centering, box shadows etc much easier to implement.

The pages still rely on information only passed if `isSnapDialog` is true, therefore we now have a separate flag for the props being passed etc and the header being visible.